yt_dlp/extractor/radioradicale.py

```diff
@@ -0,0 +1,105 @@
+            'description': '-) Votazione per l\'elezione di un giudice della Corte Costituzionale (nono scrutinio)',
+            'location': 'CAMERA',
+            'duration': 5868.0,
+            'timestamp': 1730246400,
+            'upload_date': '20241030',
+        },
+        'playlist': [{
+            'md5': 'aa48de55dcc45478e4cd200f299aab7d',
+            'info_dict': {
+                'id': '742783-0',
+                'ext': 'mp4',
+                'title': 'Parlamento riunito in seduta comune (11ª della XIX legislatura)',
+            },
+        }, {
+            'md5': 'be915c189c70ad2920e5810f32260ff5',
+            'info_dict': {
+                'id': '742783-1',
+                'ext': 'mp4',
+                'title': 'Parlamento riunito in seduta comune (11ª della XIX legislatura)',
+            },
+        }, {
+            'md5': 'f0ee4047342baf8ed3128a8417ac5e0a',
+            'info_dict': {
+                'id': '742783-2',
+                'ext': 'mp4',
+                'title': 'Parlamento riunito in seduta comune (11ª della XIX legislatura)',
+            },
+        }],
+    }]
+
+    def _entries(self, videos_info, page_id):
+        for idx, video in enumerate(traverse_obj(
+                videos_info, ('playlist', lambda _, v: v['sources']))):
+            video_id = f'{page_id}-{idx}'
+            formats = []
+            subtitles = {}
+
+            for m3u8_url in traverse_obj(video, ('sources', ..., 'src', {url_or_none})):
+                fmts, subs = self._extract_m3u8_formats_and_subtitles(m3u8_url, video_id)
+                formats.extend(fmts)
+                self._merge_subtitles(subs, target=subtitles)
+            for sub in traverse_obj(video, ('subtitles', ..., lambda _, v: url_or_none(v['src']))):
+                self._merge_subtitles({sub.get('srclang') or 'und': [{
+                    'url': sub['src'],
+                    'name': sub.get('label'),
+                }]}, target=subtitles)
+
+            yield {
+                'id': video_id,
+                'title': video.get('title'),
+                'formats': formats,
+                'subtitles': subtitles,
+            }
+
+    def _real_extract(self, url):
+        page_id = self._match_id(url)
+        webpage = self._download_webpage(url, page_id)
+
+        videos_info = self._search_json(
+            r'jQuery\.extend\(Drupal\.settings\s*,',
+            webpage, 'videos_info', page_id)['RRscheda']
+
+        entries = list(self._entries(videos_info, page_id))
+
+        common_info = {
+            'id': page_id,
+            'title': self._og_search_title(webpage),
+            'description': self._og_search_description(webpage),
+            'location': videos_info.get('luogo'),
+            **self._search_json_ld(webpage, page_id),
+        }
+
+        if len(entries) == 1:
+            return {
+                **entries[0],
+                **common_info,
+            }
```

Comment:
Is this specific for "multi_video" lists? I mean, we need to directly return the video if it is the only one in a "multi_video"list. Am I right?

Reply:
Not specific to `multi_video`, we do this for some other extractors as well. Here I think it makes sense. If a single video is on the page, return that single video, otherwise return multiple videos in a playlist.
In this specific case, the playlist is a multi video, since for the examples I found the videos are part of a larger video that just needs to be concatenated. So the example in the tests is actually a single three-hour video that was just split.

Reply:
OK. I think I may get the point.

If the webpage looks like a playlist or it contains multiple standalone media items, use `playlist`; otherwise, use `multi_video`. If there is only one video in a non-playlist webpage, just return that video itself.